data/transactions/logic/eval.go

```diff
@@ -4096,9 +4096,9 @@ func (cx *EvalContext) resolveAccount(account stackValue) (basics.Address, int,
 
 	idx, err := cx.txn.Txn.IndexByAddress(addr, cx.txn.Txn.Sender)
 	if err != nil {
-		// we don't want to convery `err`, because the supplied `account` does
+		// we don't want to convey `err`, because the supplied `account` does
 		// seem to be an address, but we can't give a valid index.
-		return addr, -1, nil
+		return addr, -1, nil //nolint:nilerr // provide -1 with nil err
```

Comment:
I guess so. The comment above is a long, involved explanation. This comment seems to repeat exactly what the `return` statement says.

Does the linter require an explanation after the `nolint` specifically?

Reply:
Are you saying we're allowed to proceed the existing comment with //noline:nilerr (instead of placing it on the same line) and have it apply to the following return?  I'd prefer that, since the explanation is exactly explaining the nilerr.

Reply:
ha, yeah, we agree